modules/ROOT/pages/Features/Patching.adoc

```diff
@@ -191,6 +191,22 @@ Here's an example schematic patch from RePan that entirely replaces the recipe u
 }
 ```
 
+Schematics can also unlock other schematics.
+Here's an example schematic patch that will automatically unlock the "Charcoal" schematic when the "Biocoal" schematic is unlock:
+
+```json
+//Game/FactoryGame/Schematics/Alternate/Parts/Schematic_Alternate_Coal1.Schematic_Alternate_Coal1_C
+{
+    "$schema": "https://raw.githubusercontent.com/budak7273/ContentLib_Documentation/main/JsonSchemas/CL_Schematic.json",
+    "ClearSchematics": false,
+    "Schematics": [
+        "Schematic_Alternate_Coal2_C"
+    ]
+}
+```
+
+Note: This will not automatically work the otherway around, you would need to patch "Schematic_Alternate_Coal2" as well if that's the behavior you wanted.
```

Comment:
```suggestion
Note that doing this does not prevent `Schematic_Alternate_Coal2_C` from being unlocked through its original means -
it simply adds an additional possible way to unlock it.
However, since `Schematic_Alternate_Coal2_C` is of type Alternate and this patch causes it to be unlocked,
`Schematic_Alternate_Coal2_C` will be removed from the alternate pool upon unlocking `Schematic_Alternate_Coal1_C`.
Unless there was already an unclaimed hard drive with `Schematic_Alternate_Coal2_C`, in which case selecting it would give you nothing new.
```